### PR TITLE
Fix : Github icon on footer is dimmer than other icons [UI} #4480

### LINKED ISSUE #4480: Github icon on footer is dimmer than other icons [UI}
### Description

The bug is in the footer of the layer5.io website. If we look closely, when all icons are inactive, github icon looks more dull (more transparent) than other icons: 
image

### Expected Behavior

The icon should be on consisten opacity with other icons

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

Please assign me this issue

### PR DESCRIPTION
**Pull request by Ykumar1415**
**Description**

This PR fixes #4480

**Notes for Reviewers**
Before : 
250661041-aa5aefe8-bde5-4902-b72b-cec13bc89378

After : 
Screenshot from 2023-07-10 21-51-10

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8ae03a535ed65cda2d7b3dc673a37eafb5a72c25 at: https://64a7e8fb420f5f5a0101203e--layer5.netlify.app

**Comment by Ykumar1415** [comment]
@leecalcote @Chadha93 @vishalvivekm can you please review this pr .

File: src/components/Case-study-banner/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
```
Comment (samyakshah3008) [inline src/components/Case-study-banner/index.js:1]: I assume in this file we were not using useEffect. @Ykumar1415 ?

**Review by samyakshah3008 — APPROVED** [review]
Great work @Ykumar1415 ✅  
Looks good to merge from my end!

File: src/components/Case-study-banner/index.js (hunk 1 above)
Comment (Ykumar1415) [reply src/components/Case-study-banner/index.js:1]: yes @samyakshah3008  that's why i removed it.

Comment (samyakshah3008) [reply src/components/Case-study-banner/index.js:1]: Sounds cool.

**Comment by vishalvivekm** [comment]
@Ykumar1415  Let's discuss this on websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit), if you would.

**Comment by Ghat0tkach** [comment]
image

changing filter from  `filter: grayscale(1) invert(0.25)`  to `filter: grayscale(1.5) invert(0.35)`  in the original footer results in the above footer .
What are your thoughts on this @Ykumar1415  and @thisiskaransgit  as it prevents  changing of current svg.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
You can't change the actual logo to fix this issue, try adjusting the CSS for the Gtihub logo specifically and not the SVG itself.

**Comment by Ghat0tkach** [comment]
> You can't change the actual logo to fix this issue, try adjusting the CSS for the Gtihub logo specifically and not the SVG itself.

He has reversed the logo change commit .😅

**Comment by Ykumar1415** [comment]
@thisiskaransgit @leecalcote @Chadha93 @vishalvivekm 

> You can't change the actual logo to fix this issue, try adjusting the CSS for the Gtihub logo specifically and not the SVG itself.

as  @Ghat0tkach already mentioned i have removed my previous changes and now made changes in styles of previous icon. Did not changed icon.

**Comment by Ykumar1415** [comment]
@leecalcote @thisiskaransgit @Chadha93 can you please review these changes. In new commit i have only changed the styles of previous github icon rather than changing complete svg.

**Comment by Nikhil-Ladha** [comment]
> @leecalcote @thisiskaransgit @Chadha93 can you please review these changes. In new commit i have only changed the styles of previous github icon rather than changing complete svg.

That's what the point is, you shouldn't be altering the styles of the SVG at all. Use CSS to update the icon color on the page where the issue is.

**Comment by Ykumar1415** [comment]
> > @leecalcote @thisiskaransgit @Chadha93 can you please review these changes. In new commit i have only changed the styles of previous github icon rather than changing complete svg.
> 
> That's what the point is, you shouldn't be altering the styles of the SVG at all. Use CSS to update the icon color on the page where the issue is.

@Nikhil-Ladha I have done the changes in img section and removed them from svg. I think now it is fine.

**Comment by l5io** [comment]
🚀 Preview for commit 6e119c6eb52e3bfa12a74fd2bb57bdef437d5954 at: https://64af9ee1b08cad2637aff518--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
> > > @leecalcote @thisiskaransgit @Chadha93 can you please review these changes. In new commit i have only changed the styles of previous github icon rather than changing complete svg.
> > 
> > 
> > That's what the point is, you shouldn't be altering the styles of the SVG at all. Use CSS to update the icon color on the page where the issue is.
> 
> @Nikhil-Ladha I have done the changes in img section and removed them from svg. I think now it is fine.

The hover effect needs to be updated as well, to change to original color.

**Comment by Ykumar1415** [comment]
> > > > @leecalcote @thisiskaransgit @Chadha93 can you please review these changes. In new commit i have only changed the styles of previous github icon rather than changing complete svg.
> > > 
> > > 
> > > That's what the point is, you shouldn't be altering the styles of the SVG at all. Use CSS to update the icon color on the page where the issue is.
> > 
> > 
> > @Nikhil-Ladha I have done the changes in img section and removed them from svg. I think now it is fine.
> 
> The hover effect needs to be updated as well, to change to original color. That said, apply the CSS to the `.social_icons a.github` class, instead of the inline style to the `img` tag.

@Nikhil-Ladha according to you what should be change in hover effact ?

File: src/components/SocialLinks-Color/index.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ const SocialLinksColor = () => {
               height="30px"
               src={github_icon}
               alt="github"
+              style={{ filter: 'grayscale(1.5) invert(0.38)' }}
```
Comment (Nikhil-Ladha) [inline src/components/SocialLinks-Color/index.js:54]: Remove the style from here, and add the CSS on [this](https://github.com/layer5io/layer5/blob/6e119c6eb52e3bfa12a74fd2bb57bdef437d5954/src/components/SocialLinks-Color/sociallinkscolor.style.js#L12) line like
```
.github {
    filter: 'grayscale(1.5) invert(0.38)';
}
```

Comment (Ykumar1415) [reply src/components/SocialLinks-Color/index.js:54]: yes i will but previously you was telling that to change something in hover effact. i don't think it is needed to change in hover  effact if it is needed then can you please specify.

**Comment by l5io** [comment]
🚀 Preview for commit 82f70b681e4f5d1eeb88b75bcd9a1c06c0cb2892 at: https://64b0aaeb5ad95968144cdb0b--layer5.netlify.app

File: src/components/SocialLinks-Color/index.js (hunk 2 above)
Comment (Ykumar1415) [reply src/components/SocialLinks-Color/index.js]: @Nikhil-Ladha i have done the complete changes that are required and suggested by you. Can you now review this pr.

**Comment by l5io** [comment]
🚀 Preview for commit 6730246e3dd17878bd5196cc2b77f2a96a75ccc1 at: https://64b0d19f8031357b1b9f950f--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Looks good now, thank you @Ykumar1415 👍🏼

**Pull request merged**